CS_Assignment_2/Question_5.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CS_Assignment_2
+{
+    internal class Question_5
+    { 
+        int emp_id;
+        string Emp_Name;
+        int Emp_Salary;
+        string Emp_Department;
+
+        public void ShowEmpData()
+        {
+            List<Question_5> employee = new List<Question_5>()
+    {
+        new Question_5{emp_id = 101, Emp_Name = "Amit",
```

Comment:
dont create object of same class. Create another class

Reply:
Another class created